src/main/sensors/acceleration_init.c

```diff
@@ -86,6 +86,14 @@
 
 #include "acceleration_init.h"
 
+#if !defined(USE_ACC_MPU6500) && !defined(USE_ACC_SPI_MPU6000) && !defined(USE_ACC_SPI_MPU6500) &&  \
```

Comment:
Need to add `if defined CLOUD_BUILD`

Reply:
Are you sure? The check is equally valid regardless.

Reply:
We need to fix SITL build.

Reply:
`SIMULATOR_BUILD` might work.